### PR TITLE
Fix code block tags

### PR DESCRIPTION
**Pull request by rubiesonthesky**
#### Summary

Fixes some incorrect code block tags. 

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by Josh-Cena — COMMENTED** [review]
Let's say: "plain" for code blocks that should be other languages (just aren't enabled), and no language tag for code blocks that actually does not represent any code? In Yari there's a comment saying marking "plain" is really subpar.

File: files/en-us/web/http/headers/content-security-policy/index.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ Reporting directives control the reporting process of CSP violations. See also t
     > while also adding forward compatibility when browsers get {{CSP("report-to")}} support,
     > you can specify both **`report-uri`** and {{CSP("report-to")}}:
     >
-    > ```html
+    > ```plain
```
Comment (Josh-Cena) [inline files/en-us/web/http/headers/content-security-policy/index.md:155]: Why isn't this HTTP? Ditto for a few ones below

**Comment by Josh-Cena** [comment]
That's just my own little wishful convention that I'm not sure anyone else follows. But rest assured that "plain" shouldn't be encouraged.

File: files/en-us/web/http/proxy_servers_and_tunneling/index.md [hunk 2]
```diff
@@ -62,17 +62,9 @@ See also the [HTTP tunnel article on Wikipedia](https://en.wikipedia.org/wiki/HT
 
 A [Proxy Auto-Configuration (PAC)](/en-US/docs/Web/HTTP/Proxy_servers_and_tunneling/Proxy_Auto-Configuration_PAC_file) file is a [JavaScript](/en-US/docs/Web/JavaScript) function that determines whether web browser requests (HTTP, HTTPS, and FTP) go directly to the destination or are forwarded to a web proxy server. The JavaScript function contained in the PAC file defines the function:
 
-The auto-config file should be saved to a file with a `.pac` filename extension:
+The auto-config file should be saved to a file with a `.pac` filename extension, `proxy.pac`.
```
Comment (jpmedley) [inline files/en-us/web/http/proxy_servers_and_tunneling/index.md:65]:
```suggestion
The auto-config file should be saved to a file with a `.pac` filename extension,: `proxy.pac`.
```

Comment (jpmedley) [reply files/en-us/web/http/proxy_servers_and_tunneling/index.md:65]: I don't believe MDN has a policy on situations such as this. I agree with your change. I personally dislike it when a sentence straddles an example.

File: files/en-us/web/http/proxy_servers_and_tunneling/proxy_auto-configuration_pac_file/index.md [hunk 3]
```diff
@@ -69,17 +69,9 @@ If all proxies are down, and there was no DIRECT option specified, the browser w
 - `PROXY w3proxy.netscape.com:8080; SOCKS socks:1080`
   - : Use SOCKS if the primary proxy goes down.
 
-The auto-config file should be saved to a file with a .pac filename extension:
+The auto-config file should be saved to a file with a .pac filename extension, `proxy.pac`.
```
Comment (jpmedley) [inline files/en-us/web/http/proxy_servers_and_tunneling/proxy_auto-configuration_pac_file/index.md:72]:
```suggestion
The auto-config file should be saved to a file with a .pac filename extension: `proxy.pac`.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
Good work here, there's just one punctuation change.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**